static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx

```diff
@@ -207,6 +211,10 @@ const StyledLoadingIndicatorWrapper = styled('div')`
   width: 100%;
   padding: ${space(2)} 0;
   height: 84px;
+
+  & + & {
```

Comment:
What is this? Whenever we do weird stuff like this, the next person to read it will have no idea what its for